app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  enum role: {student: 0, tutor: 1, admin: 2}.freeze
+end
```

Comment:
Would like to see some basic validations for `role` field and test coverage for this model